docs/testnet/onomy-testnet-docs/setting-up-a-validator.md

```diff
@@ -153,7 +158,7 @@ onomyd --home $HOME/.onomy tx bank send <your validator address> <your delegate
 ```
 2. Using faucet command, from the Onomy-side faucet:
 ```
-curl -X POST http://testnet1.onomy.io:8000/ -H  "accept: application/json" -H  "Content-Type: application/json" -d "{  \"address\": \"<your delegate cosmos address>\",  \"coins\": [    \"10000000nom\"  ]}"
+curl -X POST http://testnet1.onomy.io:8000/ -H  "accept: application/json" -H  "Content-Type: application/json" -d "{  \"address\": \"<your delegate cosmos address>\",  \"coins\": [    \"100000000nom\"  ]}"
```

Comment:
For the next testnet the main currency will be "anom", so we need to change it for all scripts and docs

Reply:
Changed, Check once